.github/workflows/merge.yml

```diff
@@ -0,0 +1,22 @@
+name: Automatically merge Dependabot PRs
+
+on:
+  check_suite:
+    types:
+      - completed
+
+jobs:
```

Comment:
One of these is responsible for `dependabot.yml` initiated PRs and the other is for GitHub Security PRs.